docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,384 @@ Risk factor mediation
+by reporting relative risks (RRs).
+To inform the direct BMI→IHD arrow, we'd need the RR of BMI
+on IHD to be adjusted for SBP, in order to isolate the direct effect.
+If we used an RR of BMI on IHD that was not adjusted for SBP, the mediated
+part of the effect would be double-counted in our model: once in the direct
+path and once in the mediated path.
+This logic extends naturally to the situation where
+there are multiple mediated pathways in addition to a direct effect:
+for the direct effect, we need an RR adjusted for **all** mediated pathways.
+
+As always when we use risk effects, whether or not mediation is involved,
+the effects should be causal; they should exclude :ref:`confounding <confounding>`.
+In an observational study, this means they should be adjusted for known
+confounders/common causes (factors affecting both our risk and our outcome),
+**in addition** to adjusting for the modeled mediator when calculating the direct RR.
+Crucially, in this case, the RR of SBP→IHD **must be adjusted for BMI** since
+that is a confounder of this effect.
+
+Note that while the ideal would be evidence from a randomized controlled trial,
+therefore controlling for *all* confounders (known and unknown), that frequently isn't possible,
+including in this example:
+there isn't a way to randomly assign people BMIs or SBPs since these variables
+are not directly manipulable.
+
+GBD-modeled mediation
++++++++++++++++++++++
+
+.. todo::
+  This section only covers mediation methods from GBD 2019.
+  It appears that substantial aspects of mediation may change in GBD 2021.
+  We have yet to model any mediation using GBD 2021 but will need to expand this
+  to cover new methods.
+
+In a mediation situation where the primary risk and mediator are both GBD-modeled risks
+and the outcome is a GBD-modeled cause, GBD may report a "mediation factor" for the pathway.
+(If it does not, that means that one or more of the causal effects in the mediation diagram
+were not modeled by GBD.)
+Mediation factors for GBD 2019 and 2021 can be found `on this HUB page <https://hub.ihme.washington.edu/display/GBD2020/GBD+2021+Risk+factors+mediation>`_.
+GBD mediation factors do not vary by location, age, sex, or year.
+They are not allowed to exceed 1; if the calculated value is greater than 1 it
+will be truncated to exactly 1 (this truncation happens at the draw level).
+Some mediation pathways have a mediation factor of exactly 1 by definition, because
+there is no direct effect.
+For example, lead exposure's effect on ischemic heart disease is entirely mediated by
+high systolic blood pressure.
+Draws of the mediation factor are allowed to be negative, which represents the direct pathway
+and the mediated pathway having different effect directions.
+However, if more than 50 of the 1,000 draws were negative, that mediation
+pathway was excluded from GBD.
+
+.. todo::
+  Why does GBD exclude due to negative draws?
```

Comment:
I'm guessing that it is because this mediation is not "significant" at the 0.05 level

Reply:
Yeah, but the way GBD states it, I think they would exclude even if if 970/1,000 draws were negative (a "significant" effect with a negative direction). I'll clarify this and turn it into a note.